react/features/app/components/AbstractApp.js

```diff
@@ -28,14 +30,13 @@ export class AbstractApp extends Component {
      * @static
      */
     static propTypes = {
-        config: React.PropTypes.object,
```

Comment:
@BeatC, I don't like that you removed the `config` prop from the public API of the React component `App`. I'd still like consumers of the public API to be able to provide their own `config` prop value. Let's please talk before you continue with modifications to this PR.